### ISSUE TITLE
Hypoechoic nodule

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10113]
**Preferred term label:**
Hypoechoic nodule

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C4720860

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lobular breast carcinoma

**Your nano-attribution (ORCID)**
